src/Sentry.Maui.CommunityToolkitMvvm/MauiAppBuilderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using Sentry.Maui.CommunityToolkitMvvm;
+
+namespace Sentry.Maui;
+
+/// <summary>
+/// Methods to hook into MAUI & CommunityToolkit.Mvvm
+/// </summary>
+public static class MauiAppBuilderExtensions
+{
+    /// <summary>
+    /// Installs necessary services to auto-instrument CommunityToolkit.Mvvm commands
+    /// </summary>
+    /// <param name="builder">The MauiAppBuilder</param>
+    /// <returns>The MauiAppBuilder</returns>
+    public static MauiAppBuilder UseSentryCommunityToolkitIntegration(this MauiAppBuilder builder)
+    {
+        builder.Services.UseSentryCommunityToolkitIntegration();
+        return builder;
+    }
+
+
+    /// <summary>
+    /// Installs necessary services to auto-instrument CommunityToolkit.Mvvm commands
+    /// </summary>
+    /// <param name="services">The Service Collection</param>
+    /// <returns>The Service Collection</returns>
+    public static IServiceCollection UseSentryCommunityToolkitIntegration(this IServiceCollection services)
```

Comment:
Do we expect folks to call this one? Seems like the public API is just `builder.UseSentryCommunityToolkitIntegration()` which calls this one, so we could hide it from the user:


```suggestion
    internal static IServiceCollection UseSentryCommunityToolkitIntegration(this IServiceCollection services)
```

Reply:
Because this is an external integration/secondary package, I can't install it without the user acting on it in their code.  We could consider "help" with this in the future using source generators.